tools/pg_describe

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env node
+
+const async = require('async');
+const ERR = require('async-stacktrace');
+const colors = require('colors');
+const fs = require('fs-extra');
+const _ = require('lodash');
+const path = require('path');
+
+const databaseDescribe = require('../tests/databaseDescribe');
+
+const yargs = require('yargs')
```

Comment:
Maybe we should add a `.strict()` to `yargs`? I was bitten yesterday by misspelling an argument and it was silently ignored rather than giving an error.